src/client/camera.ts

```diff
@@ -181,7 +186,7 @@ class Camera {
     ctx.fill();
 
     ctx.drawImage(
-      textures.unit[unit.type] as CanvasImageSource,
+      (textures.unit[unit.type] || textures.missing) as CanvasImageSource,
```

Comment:
Should this be changed to `?? textures.missing` just to be more consistent with the rest of the codebase?
(Same applies to lines `243` and `353` of this file)

Reply:
Yes, my bad.